ext/gettext/gettext.stub.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/** @return string|false */
+function textdomain(?string $domain = null) {}
```

Comment:
This parameter shouldn't have a default: While it accepts `null`, it needs to be explicitly passed.

Reply:
Thanks, fixed now